diff --git a/node_modules/sortablejs/modular/sortable.esm.js b/node_modules/sortablejs/modular/sortable.esm.js
index ba685cb..f3d3bcd 100644
--- a/node_modules/sortablejs/modular/sortable.esm.js
+++ b/node_modules/sortablejs/modular/sortable.esm.js
@@ -2808,7 +2808,10 @@ var autoScroll = throttle(function (evt, options, rootEl, isFallback) {
   if (!options.scroll) return;
   var x = (evt.touches ? evt.touches[0] : evt).clientX,
       y = (evt.touches ? evt.touches[0] : evt).clientY,
-      sens = options.scrollSensitivity,
+      sensTop = typeof options.scrollSensitivity === 'object' ? options.scrollSensitivity.top : options.scrollSensitivity,
+      sensBottom = typeof options.scrollSensitivity === 'object' ? options.scrollSensitivity.bottom : options.scrollSensitivity,
+      sensLeft = typeof options.scrollSensitivity === 'object' ? options.scrollSensitivity.left : options.scrollSensitivity,
+      sensRight = typeof options.scrollSensitivity === 'object' ? options.scrollSensitivity.right : options.scrollSensitivity,
       speed = options.scrollSpeed,
       winScroller = getWindowScrollingElement();
   var scrollThisInstance = false,
@@ -2853,8 +2856,8 @@ var autoScroll = throttle(function (evt, options, rootEl, isFallback) {
       canScrollY = height < scrollHeight && (elCSS.overflowY === 'auto' || elCSS.overflowY === 'scroll');
     }
 
-    var vx = canScrollX && (Math.abs(right - x) <= sens && scrollPosX + width < scrollWidth) - (Math.abs(left - x) <= sens && !!scrollPosX);
-    var vy = canScrollY && (Math.abs(bottom - y) <= sens && scrollPosY + height < scrollHeight) - (Math.abs(top - y) <= sens && !!scrollPosY);
+    var vx = canScrollX && (Math.abs(right - x) <= sensRight && (scrollPosX + width) < scrollWidth) - (Math.abs(left - x) <= sensLeft && !!scrollPosX);
+    var vy = canScrollY && (Math.abs(bottom - y) <= sensBottom && (scrollPosY + height) < scrollHeight) - (Math.abs(top - y) <= sensTop && !!scrollPosY);
 
     if (!autoScrolls[layersOut]) {
       for (var i = 0; i <= layersOut; i++) {
